server/engine/src/main/java/org/eclipse/lsp/cobol/core/engine/dialects/v2/DialectErrorHelper.java

```diff
@@ -34,16 +33,14 @@
 @UtilityClass
 class DialectErrorHelper {
 
-  public SyntaxError processingError(
-      MessageService messageService, Locality locality, String dialect, String message) {
+  public SyntaxError processingError(Locality locality, String dialect, String message) {
     return SyntaxError.syntaxError()
         .errorSource(ErrorSource.DIALECT)
         .location(locality.toOriginalLocation())
         .messageTemplate(
             MessageTemplate.of(
                 "GrammarPreprocessorListener.errorProcessingDialect", dialect, message))
         .severity(ERROR)
-        .errorCode(ErrorCodes.MISSING_COPYBOOK)
```

Comment:
Removing this might break the diagnostic filtering.

Reply:
It is an error, this must not be here